universal_transfer_operator/docs/supported_transfer_mode.rst

```diff
@@ -0,0 +1,10 @@
+.. _supported_transfer_modes:
+
+Supported Transfer modes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+With universal transfer operator, users can perform data transfers using the following transfer modes:
```

Comment:
I think it would be nice if we can link it to concepts, explaining the transfers.